src/integration/java/io/pinecone/integration/controlPlane/pod/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -33,7 +33,7 @@ public void describeAndListIndex() {
         assertNotNull(indexModel);
         assertEquals(indexDimension, indexModel.getDimension());
         assertEquals(indexName, indexModel.getName());
-        assertEquals(IndexMetric.COSINE, indexModel.getMetric());
+        assertEquals(IndexMetric.DOTPRODUCT, indexModel.getMetric());
```

Comment:
Same Q as in the other file -- why dotproduct here again?

Reply:
Indexes must be dotproduct in order to allow querying sparse vectors: https://docs.pinecone.io/guides/data/understanding-hybrid-search#limitations

Most of our data plane tests use sparse vectors in some way so we need our indexes to be dotproduct.